src/runtime/hexagon_remote/Makefile

```diff
@@ -152,6 +153,13 @@ bin/%/sim_host.o: sim_host.cpp sim_protocol.h
 	mkdir -p $(@D)
 	$(CXX-$*) -std=c++11 $(CCFLAGS-$*) -c sim_host.cpp -o $@
 
+bin/%/sim_qurt/sim_qurt.o: sim_qurt.cpp
+	mkdir -p $(@D)
+	$(CXX-$*) $(CCFLAGS-$*) -O3 -c sim_qurt.cpp -o $@
```

Comment:
minor: -O3 is part of COMMON_FLAGS which is part of CCFLAGS-v60

Reply:
Corrected